components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <code class="algorithms__code-cell">
+          <pre class="algorithms__code-cell__line">pip install qiskit</pre>
+          <pre class="algorithms__code-cell__line">pip install qiskit-ibm-provider</pre>
+        </code>
+        <!--
+          @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+        -->
+        <cv-button
+          title="Copy"
+          size="sm"
+          class="algorithms__copy-button"
+          @click="copyToClipboard($event)"
+        >
+          Copy
+        </cv-button>
+      </div>
+      <h3>Test some algorithms</h3>
+      <cv-tabs container>
+        <cv-tab v-for="algorithm in algorithmsContent" :key="algorithm.title" :label="algorithm.title">
+          <div class="algorithms__section">
+            <code class="algorithms__code-cell">
+              <template v-for="(line, index) in algorithm.codeSample">
+                <!-- eslint-disable vue/no-v-html -->
+                <pre v-if="line.startsWith('#')" :key="line" class="algorithms__code-cell__comment" v-html="line" />
+                <pre v-else :key="index" class="algorithms__code-cell__line" v-html="line" />
+                <!-- eslint-enable -->
+              </template>
+            </code>
+            <cv-button
+              title="Copy"
+              size="sm"
+              class="algorithms__copy-button"
+              @click="copyToClipboard($event)"
+            >
+              Copy
+            </cv-button>
+          </div>
```

Comment:
This code also could be replace with the `CodeSnippet` component

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3020/commits/d60f48b9a31ffecbe3e17f577ebfa01892204e93